### DISCUSSION TITLE
Welcome to electron-shadcn Discussions!

### DISCUSSION THREAD

**Discussion by LuanRoger** [LuanRoger/electron-shadcn#52] [Announcements]
## 👋 Welcome!

Here you can:

- 📣 Check the latest changes and major updates.
- 🗳️ Vote for changes.
- 🤝 Ask for help
- 🙌 Show off all the amazing things you have made with electron-shadcn.

**Comment by maxwhipw** [comment]
Thanks for making this man! This seems to be one of the most beloved combination of js tools out there but I haven't seen it all put together in a well maintained way. It is a big blessing. Will post app here when it's further along.

**Comment by qza2468** [comment]
really thanks, read a lot, try lots, only you save me.

**Reply by LuanRoger** [reply]
Thanks.

I'm planning to open a pool to vote for new tools to be part of the template, beside other changes. Probably, the next thing I will do is switch from Prettier/ESLint to Biome.

**Reply by maxwhipw** [reply]
> Thanks.
> 
> I'm planning to open a pool to vote for new tools to be part of the template, beside other changes. Probably, the next thing I will do is switch from Prettier/ESLint to Biome.

+1 to swtiching to Biome. I really don't like Prettier or ESLint. ESLint appears to be a headache to configure even for veterans and Prettier I'm not happy that it removes extra spacing between lines of code without a way to disable.

**Reply by LuanRoger** [reply]
> > Thanks.
> > I'm planning to open a pool to vote for new tools to be part of the template, beside other changes. Probably, the next thing I will do is switch from Prettier/ESLint to Biome.
> 
> +1 to swtiching to Biome. I really don't like Prettier or ESLint. ESLint appears to be a headache to configure even for veterans and Prettier I'm not happy that it removes extra spacing between lines of code without a way to disable.

Poll open: https://github.com/LuanRoger/electron-shadcn/discussions/400

This will help me to understand the developers needs with the template to keep it relevant.
